pkg/signals/signal_posix.go

```diff
@@ -0,0 +1,23 @@
+// +build !windows
```

Comment:
You don't need to copy these files - you can reuse from controller-runtime - see https://github.com/submariner-io/lighthouse/pull/88.